whylog/assistant/regex_assistant/regex_object.py

```diff
@@ -8,69 +10,80 @@ class RegexObject(object):
     Supports interactive updates like adding/removing groups to regex, replacing regex
```

Comment:
The docsString is outdated. This class doesn't have functions for adding and removing groups to regex.